docs/source/usage.rst

```diff
@@ -0,0 +1,54 @@
+Usage
+#####
+
+Basic usage with raster files
+=============================
+
+
+
+.. code-block:: python
+
+    from tophu import RasterBand, SnaphuUnwrap, multiscale_unwrap
+    import rasterio as rio
+    # create `RasterBand`s from the input interferogram and coherence
```

Comment:
Can you consistently capitalize the first letter of each comment and end comments with a period throughout? Thanks!

```suggestion
    # Create `RasterBand`s from the input interferogram and coherence.
```